NuGet.Config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
```

Comment:
What is the purpose of this?

Reply:
https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied - it is used to `ensure that only the sources defined below are used`.